mkdocs.yml

```diff
@@ -19,7 +19,7 @@ nav:
 - Registration: 'common/registration.md'
 - Site Verification: 'site-verification.md'
 - Site Maintenance: 'site-maintenance.md'
-- Help: 'common/help.md'
+- 'Help / Security Incidents': 'common/help.md'
```

Comment:
How does this look in the ToC? Maybe just `Help/Security Incidents` if that prevents overflow of this item across two lines